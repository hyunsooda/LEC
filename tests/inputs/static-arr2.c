#include <stdio.h>

int main() {
    const int size = 5;
    int arr[size];
    printf("%d\n", arr[13]);
}
