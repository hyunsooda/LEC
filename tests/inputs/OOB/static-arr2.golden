B[31mFound out of bound access: ["static-arr2.c":6:20]B[37m
    array length: 1, indexed by: 13
    variable name: "arr", allocated at: 5
